src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -311,12 +383,24 @@ export default {
 		getFocusableList() {
 			return this.$el.querySelectorAll('li.acli_wrapper .acli')
 		},
+		setIsFocused(event) {
+			if (event.relatedTarget?.className.includes('input-field__clear-button') || this.searchText !== '') {
```

Comment:
Like that behaviour, when blurred input stays expanded with text. Make sense, as we don't have filters applied at the moment
![image](https://github.com/nextcloud/spreed/assets/93392545/f42cde07-ade5-4975-a3ba-0c4f6322bf91)